specification/electronic-prescription-service-api.yaml

```diff
@@ -296,34 +298,87 @@ paths:
         '4XX':
           description: |
             An error occurred as follows:
-            
-            | HTTP status | Error code         | Description                                    | FHIR type        |
-            | ----------- | ------------------ | ---------------------------------------------- | -----------------|
+
+            | HTTP status | Error code         | Description                                     | FHIR type        |
+            | ----------- | ------------------ | ----------------------------------------------- | ---------------- |
             | 400         | BAD_REQUEST        | Spine Error - duplicate prescription ID exists. | OperationOutcome |
             | 400         | BAD_REQUEST        | Spine Error - prescription not found.           | Bundle           |
           content:
             application/fhir+json:
               schema:
-                type: object
+                $ref: '#/components/responses/200OperationOutcome'
               examples:
                 OperationOutcome:
                   value:
                     $ref: components/examples/example-a-validation-error-missing-field/Response-FhirError.json
                 Bundle:
                   value:
                     $ref: components/examples/errors/prescription-order-update/bundle-error.json
+  /FHIR/R4/Task:
+    post:
+      operationId: return / withdraw
+      summary: Send either a Task-dispense-notification-amend or Task-dispense-proposal-return to EPS
+      description: |
+        ## Overview
+        This endpoint can be used to return a downloaded prescription to Spine, so that another dispensing system can download it. 
+        
+        It can also be used to withdraw a dispensed prescription to Spine, for example in the case that a prescription was marked as dispensed erroneously.
```

Comment:
Not sure the first bit of this sentence makes sense.

It can also be used to withdraw a dispense notification from Spine?